--- kdeartwork-030329/kscreensaver/kdesavers/Makefile.am~	Tue Dec 10 00:52:30 2002
+++ kdeartwork-030329/kscreensaver/kdesavers/Makefile.am	Tue Apr  1 11:25:29 2003
@@ -57,6 +57,6 @@
                KBlob.desktop KVm.desktop KMatrix.desktop KScience.desktop\
                KClock.desktop $(GL_DESKTOP_FILES) $(KCLOCK_DESKTOP_FILE)
 
-desktopdir = $(kde_appsdir)/System/ScreenSavers
+desktopdir = $(kde_datadir)/kscreensaver
 
 EXTRA_DIST = $(desktop_DATA) vm.xpm vm.xbm kscience.png  particle.png image.png
--- kdeartwork-030329/kscreensaver/kpartsaver/Makefile.am~	Tue Apr  1 10:25:29 2003
+++ kdeartwork-030329/kscreensaver/kpartsaver/Makefile.am	Tue Apr  1 11:26:56 2003
@@ -10,7 +10,7 @@
 METASOURCES = AUTO
 
 desktop_DATA = kpartsaver.desktop
-desktopdir = $(kde_appsdir)/System/ScreenSavers
+desktopdir = $(kde_datadir)/kscreensaver
 
 messages: rc.cpp
 	$(XGETTEXT) *.cpp -o $(podir)/kpartsaver.pot
--- kdeartwork-030329/kscreensaver/kxsconfig/Makefile.am~	Sat Oct  5 01:03:32 2002
+++ kdeartwork-030329/kscreensaver/kxsconfig/Makefile.am	Tue Apr  1 11:31:03 2003
@@ -40,7 +40,7 @@
 	    fi; \
 	done
 
-screensaverdir=$(kde_appsdir)/System/ScreenSavers
+screensaverdir=$(kde_datadir)/kscreensaver
 
 install-data-local:
 	$(mkinstalldirs) $(DESTDIR)$(screensaverdir)
--- kdeartwork-030914/kscreensaver/xsavers/Makefile.am~	2003-09-11 02:03:55.000000000 +0200
+++ kdeartwork-030914/kscreensaver/xsavers/Makefile.am	2003-09-16 18:28:07.000000000 +0200
@@ -30,7 +30,7 @@
 ########## Meta objects ##########
 
 desktop_DATA = KSpace.desktop KSwarm.desktop
-desktopdir = $(kde_appsdir)/System/ScreenSavers
+desktopdir = $(kde_datadir)/kscreensaver
 
 EXTRA_DIST = $(desktop_DATA)
 
