--- kdeartwork-3.0.3/kscreensaver/kdesavers/banner.cpp--	2002-08-29 08:11:06.000000000 +0200
+++ kdeartwork-3.0.3/kscreensaver/kdesavers/banner.cpp	2002-08-29 08:16:52.000000000 +0200
@@ -332,7 +332,7 @@ void KBannerSetup::slotOkPressed()
 
 void KBannerSetup::slotAbout()
 {
-	KMessageBox::about(this,
+  KMessageBox::information(this,
 			     i18n("Banner Version 2.2.0\n\nwritten by Martin R. Jones 1996\nmjones@kde.org\nextended by Alexander Neundorf 2000\nalexander.neundorf@rz.tu-ilmenau.de\n"));
 }
 
--- kdeartwork-3.0.3/kscreensaver/kdesavers/blob.cpp--	2002-08-29 08:11:12.000000000 +0200
+++ kdeartwork-3.0.3/kscreensaver/kdesavers/blob.cpp	2002-08-29 08:16:29.000000000 +0200
@@ -547,7 +547,7 @@ void KBlobSetup::slotOkPressed()
 
 void KBlobSetup::slotAbout()
 {
-	KMessageBox::about(this,
+	KMessageBox::information(this,
 			     i18n("Blobsaver Version 0.1\n\nwritten by Tiaan Wessels 1997\ntiaan@netsys.co.za"));
 	if (saver)
 		saver->setAlgorithm(algs->currentItem());
--- kdeartwork-3.0.3/kscreensaver/kdesavers/lines.cpp--	2002-08-29 08:11:18.000000000 +0200
+++ kdeartwork-3.0.3/kscreensaver/kdesavers/lines.cpp	2002-08-29 08:17:06.000000000 +0200
@@ -264,7 +264,7 @@ void kLinesSetup::slotColend(const QColo
 }
 
 void kLinesSetup::slotAbout(){
-	KMessageBox::about(this,
+	KMessageBox::information(this,
 		i18n("Lines Version 2.2.0\n\n"
 				   "written by Dirk Staneker 1997\n"
 				   "dirk.stanerker@student.uni-tuebingen.de"));
--- kdeartwork-3.0.3/kscreensaver/kdesavers/lorenz.cpp--	2002-08-29 08:11:24.000000000 +0200
+++ kdeartwork-3.0.3/kscreensaver/kdesavers/lorenz.cpp	2002-08-29 08:17:18.000000000 +0200
@@ -242,7 +242,7 @@ void KLorenzSetup::slotXRot(int num)
 
 void KLorenzSetup::slotAbout()
 {
-    KMessageBox::about(this,i18n("Lorenz Attractor screen saver for KDE\n\nCopyright (c) 2000 Nicolas Brodu"));
+    KMessageBox::information(this,i18n("Lorenz Attractor screen saver for KDE\n\nCopyright (c) 2000 Nicolas Brodu"));
 }
 
 // Ok pressed - save settings and exit
--- kdeartwork-3.0.3/kscreensaver/kdesavers/science.cpp--	2002-08-29 08:11:35.000000000 +0200
+++ kdeartwork-3.0.3/kscreensaver/kdesavers/science.cpp	2002-08-29 08:17:34.000000000 +0200
@@ -1182,6 +1182,6 @@ void KScienceSetup::slotOkPressed()
 void KScienceSetup::slotAbout()
 {
 	QString about = i18n("Science Version 0.26.5\n\nwritten by Rene Beutler (1998)\nrbeutler@g26.ethz.ch");
-	KMessageBox::about(this,
+	KMessageBox::information(this,
 	                      about);
 }
