--- kdeartwork-3.2.0/kscreensaver/kxsconfig/configure.in.in.orig	Sun Oct 26 11:45:03 2003
+++ kdeartwork-3.2.0/kscreensaver/kxsconfig/configure.in.in	Fri Feb  6 11:18:00 2004
@@ -32,6 +32,7 @@
      XSCREENSAVER_HACKS_DIR=`echo $XSCREENSAVER | sed -e 's,/flame$,,'`
      AC_DEFINE_UNQUOTED(XSCREENSAVER_HACKS_DIR, "$XSCREENSAVER_HACKS_DIR", [Defines where xscreensaver stores its graphic hacks])
 
+     if test -z "$XSCREENSAVER_CONFIG"; then
      AC_MSG_CHECKING([for flame.xml])
      AC_FIND_FILE(lib/xscreensaver/config/flame.xml, [ $xscreensaver_prefixes ], XSCREENSAVER_CONFIG)
      if test "$XSCREENSAVER_CONFIG" = NO; then
@@ -61,6 +62,7 @@
      else
 	XSCREENSAVER_CONFIG="$XSCREENSAVER_CONFIG/lib/xscreensaver/config"
      fi
+     fi
      AC_MSG_RESULT([$XSCREENSAVER_CONFIG])
      AC_DEFINE_UNQUOTED(XSCREENSAVER_CONFIG_DIR, "$XSCREENSAVER_CONFIG", 
 	[Defines where xscreensaver stores it's config files])
