--- kdeartwork-3.1.2/kscreensaver/kdesavers/Makefile.am~	2003-05-13 20:14:00.000000000 +0200
+++ kdeartwork-3.1.2/kscreensaver/kdesavers/Makefile.am	2003-05-13 21:06:43.000000000 +0200
@@ -6,7 +6,7 @@
 
 INCLUDES = $(all_includes) $(GLINC)
 LDFLAGS = $(all_libraries) $(KDE_RPATH)
-LDADD = $(LIB_KDEUI) -lkscreensaver -lm $(GLLIB)
+LDADD = $(LIB_KDEUI) -lkscreensaver -lGL -lGLU -lm $(GLLIB)
 
 if COMPILE_GL_KSAVERS
 GL_KSAVERS=kfountain.kss kwave.kss kgravity.kss kflux.kss keuphoria.kss ksolarwinds.kss
