--- kdeartwork-3.3.91/kscreensaver/kxsconfig/configure.in.in.orig	2004-07-06 14:53:17.000000000 +0200
+++ kdeartwork-3.3.91/kscreensaver/kxsconfig/configure.in.in	2005-01-24 14:56:31.219251592 +0100
@@ -44,6 +44,7 @@
      XSCREENSAVER_HACKS_DIR=`echo $XSCREENSAVER | sed -e 's,/flame$,,'`
      AC_DEFINE_UNQUOTED(XSCREENSAVER_HACKS_DIR, "$XSCREENSAVER_HACKS_DIR", [Defines where xscreensaver stores its graphic hacks])
 
+     if test -z "$XSCREENSAVER_CONFIG"; then
      AC_MSG_CHECKING([for flame.xml])
      for subdir in lib/xscreensaver/config \
 		   lib/X11/xscreensaver/config \
@@ -57,6 +58,7 @@
 		break
          fi
      done
+     fi
      AC_MSG_RESULT([$XSCREENSAVER_CONFIG])
      AC_DEFINE_UNQUOTED(XSCREENSAVER_CONFIG_DIR, "$XSCREENSAVER_CONFIG", 
 	[Defines where xscreensaver stores it's config files])
